Mmh I still can't pre-deploy it. `docker build .` on the server gives me this error:
```
Step 1/16 : FROM ocaml/opam2:alpine
 ---> 8029c38a73b0
Step 2/16 : RUN sudo apk add --update docker
 ---> Using cache
 ---> 27a886eab6ac
Step 3/16 : RUN cd /home/opam/opam-repository && git pull origin master && opam update -uy
 ---> Using cache
 ---> e29b3a0e2550
Step 4/16 : RUN opam switch 4.07
 ---> Using cache
 ---> 391027630a16
Step 5/16 : ENV OPAMSOLVERTIMEOUT=600
 ---> Running in 8b7d564284fb
Removing intermediate container 8b7d564284fb
 ---> 97da8f725fb0
Step 6/16 : RUN opam depext -uivy irmin-unix ezjsonm bos ptime fmt datakit-ci conf-libev
 ---> Running in f8e7bc20c896
# Detecting depexts using vars: arch=x86_64, os=linux, os-distribution=alpine, os-family=alpine
[ERROR] Sorry, resolution of the request timed out.
        Try to specify a simpler request, use a different solver, or increase the allowed time by setting OPAMSOLVERTIMEOUT to a bigger value (currently, it is set to 600.0 seconds).
Command failed: opam list --readonly --external  --resolve=irmin-unix,ezjsonm,bos,ptime,fmt,datakit-ci,conf-libev returned 60
The command '/bin/sh -c opam depext -uivy irmin-unix ezjsonm bos ptime fmt datakit-ci conf-libev' returned a non-zero code: 60
```
I guess if I continue to increase `OPAMSOLVERTIMEOUT` it might make it pass at some point. Let's try.
(cc @AltGr)

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,7 +6,7 @@ services:
 env:
   global:
   - PACKAGE="mirage-ci"
-  - EXTRA_ENV="OPAMSOLVERTIMEOUT=600"
+  - EXTRA_ENV="OPAMSOLVERTIMEOUT=6000"
   matrix:
   - OCAML_VERSION="4.06" DISTRO="alpine"
   - OCAML_VERSION="4.07" DISTRO="alpine"

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,6 @@
 language: c
 install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docker.sh
-script: bash -ex .travis-docker.sh
+script: travis_wait 30 bash -ex .travis-docker.sh
 services:
 - docker
 env:

--- a/Dockerfile
+++ b/Dockerfile
@@ -2,7 +2,7 @@ FROM ocaml/opam2:alpine
 RUN sudo apk add --update docker
 RUN cd /home/opam/opam-repository && git pull origin master && opam update -uy
 RUN opam switch 4.07
-ENV OPAMSOLVERTIMEOUT=600
+ENV OPAMSOLVERTIMEOUT=6000
 RUN opam depext -uivy irmin-unix ezjsonm bos ptime fmt datakit-ci conf-libev 
 ADD --chown=1000 mirage-ci.opam /home/opam/src/
 RUN opam install --deps-only /home/opam/src/

--- a/src/opam_docker.ml
+++ b/src/opam_docker.ml
@@ -94,6 +94,8 @@ module Cmds = struct
     from ~tag:distro "ocaml/opam2" @@
     add_cache_dir @@
     run "opam switch %s" (Oversions.to_string ocaml_version) @@
+    run "git pull origin master" @@
+    run "opam update" @@
     run "opam install -yv opam-depext%s"
       (if Oversions.older_than_4_06 ocaml_version
        then " ocaml-secondary-compiler" (* NOTE: This is needed since dune 2.0.0

